src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -107,9 +110,19 @@ export default Vue.extend<Data, Methods, unknown, unknown>({
             workflowRunStatus: null,
             error: "",
             defaultMessage: "",
+            pollingTimer: null
         };
     },
     methods: {
+        startPolling() {
+            if (!this.pollingTimer) {
+                this.pollingTimer = setInterval(() => this.getWorkflowRunStatus(this.selectedWorkflowKey), 1500);
+            }
+        },
+        stopPolling() {
+            clearInterval(this.pollingTimer)
+            this.pollingTimer = null
```

Comment:
You could do this, to make it clear that `pollingTimer` may already be null - e.g. if called from `beforeDestroy()`:
```suggestion
            if (this.pollingTimer) {
                clearInterval(this.pollingTimer)
                this.pollingTimer = null
            }
```
But as it happens it isn't an issue because `clearInterval()` [ignores invalid handles](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-clearinterval).